vembrane/__init__.py

```diff
@@ -29,7 +31,7 @@ def get_annotation_keys(header: VariantHeader) -> List[str]:
             return list(map(str.strip, rec.get("Description").split("'")[1].split("|")))
     return []
 
-
+@lru_cache
```

Comment:
Good idea! Should have been in a different PR, but whatever.
My only concerns are:
 - Default bound for the cache (how many entries are cached til old ones get ditched?)
 - Did you check if there even are identical annotation strings "in the wild", i.e. if this actually caches anything useful?

Reply:
default is 128, i thought it would be enough. We could also reduce it to 32.
i introduced the lru cache, because statistics parse_annotation_entry is called twice now. In "statistics" and "filter_vcf". In theory we could manage this in a different way and somehow pass the split records to statistics, but I think they way is is currently implemented, is the most pythonic way.